File: Code/Components/DanpheEMR.ServerModel/SystemAdminModels/InvoiceDetailsModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class InvoiceDetailsModel
    {

        public string Fiscal_Year { get; set; }
        public string Bill_No { get; set; }
        public string Customer_name { get; set; }
        public string PANNumber { get; set; }
        public string BillDate { get; set; }
        public string BillType { get; set; }
        public double Amount { get; set; }
        public double DiscountAmount { get; set; }
        public double Taxable_Amount { get; set; }
        public double Tax_Amount { get; set; }
        public double Total_Amount { get; set; }
        public string SyncedWithIRD { get; set; }
        public string Is_Printed { get; set; }
        public string Printed_Time { get; set; }
        public string Entered_by { get; set; }
        public string Printed_by { get; set; }
        public int Print_Count { get; set; }
        public string Is_Bill_Active { get; set; }//added: sud-9May'18
        public string Is_Realtime { get; set; }//added: sud-9May'18
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/system-admin/shared/invoice-details.model.ts
import {
  NgForm,
  FormGroup,
  FormControl,
  Validators,
  FormBuilder,
  ReactiveFormsModule
} from '@angular/forms';
import * as moment from 'moment/moment';
export class InvoiceDetailsModel {
  public Fiscal_Year: string = null;
  public Bill_No: number = 0;
  public Customer_name: string = null;
  public PANNumber: string = null;
  public BillDate: string = null;
  public BillType: string = null;  //now we don't show billType
  public Amount: number = 0;
  public DiscountAmount: number = 0;
  public Taxable_Amount: number = 0;
  public Tax_Amount: number = 0;
  public Total_Amount: number = 0;
  public SyncedWithIRD: string = null;
  public Is_Printed: string = null;
  public Printed_Time: string = null;
  public Entered_by: string = null;
  public Printed_by: string = null;
  public Print_Count: number = null;
  public fromDate: string = "";
  public toDate: string = "";
  public Bill_No_Str: string = "";
  public NonTaxable_Amount: number = 0;
  public Payment_Method: string = null;
  public TransactionId: number = 0;

}
